### PR TITLE
Fix crash on invalid Swift language version

### PR DESCRIPTION
**Pull request by neonichu**
Throw an error instead.

rdar://75936643

### CODE REVIEW

**Comment by neonichu** [comment]
@swift-ci please smoke test

**Review by abertelrud — APPROVED** [review]

**Comment by neonichu** [comment]
@swift-ci please smoke test linux

**Review by tomerd — APPROVED** [review]

File: Sources/PackageLoading/PackageDescription4Loader.swift [hunk 1]
```diff
@@ -71,7 +71,12 @@ enum ManifestJSONParser {
         }
 
         return try versionJSON.map {
-            try SwiftLanguageVersion(string: String(json: $0))!
+            let languageVersionString = try String(json: $0)
+            if let languageVersion = SwiftLanguageVersion(string: languageVersionString) {
+                return languageVersion
+            } else {
+                throw ManifestParseError.runtimeManifestErrors(["invalid Swift language version: \(languageVersionString)"])
+            }
```
Comment (tomerd) [inline Sources/PackageLoading/PackageDescription4Loader.swift:79]: nit: use guard?

**Comment by neonichu** [comment]
@swift-ci please smoke test

**Pull request merged**